src/main/java/com/commercetools/project/sync/Syncer.java

```diff
@@ -100,10 +100,12 @@ public Syncer(
    * this sync.
    *
    * @param runnerName the name of the sync runner.
+   * @param isFullSync whether to run a delta sync (based on the last sync timestamp) or a full
+   *     sync.
    * @return completion stage containing no result after the execution of the sync process and
    *     logging the result.
    */
-  public CompletionStage<Void> sync(@Nullable final String runnerName) {
+  public CompletionStage<Void> sync(@Nullable final String runnerName, final boolean isFullSync) {
```

Comment:
for the next changes, I could be nice to move CLI params to a context object like SyncContext/SyncOptions after 2 params which could be created from `CLIRunner#processSyncOptionAndExecute` method, I think it will be more easy to manage. But this is not needed now.

Reply:
Good idea :)